### PR TITLE
Feature/scott/#11 add bootstrap

### LINKED ISSUE #11: Add bootstrap
Make it pretty

### PR DESCRIPTION
**Pull request by Scott-Larsen**
Add [bootstrap](https://getbootstrap.com/) functionality to the app - allowing us to quickly add style to the website with only a little bit of code. Closes #11

Here's a little before and after (thx for the suggestion, @SalMireles) that doesn't show much difference but as we add content it should look better than the un-styled original.

Before:
Screen Shot 2022-03-12 at 9 00 08 PM

After:
Screen Shot 2022-03-12 at 8 51 32 PM

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -9,6 +9,7 @@ python = "==3.8.10"
 Flask = "==2.0.3"
 python-dotenv = "==0.19.2"
 Flask-WTF = "==1.0.0"
+Flask-Bootstrap = "^3.3.7"
```
Comment (SalMireles) [inline pyproject.toml:12]: Can you pin this? Also, I ran poetry install and received a warning about an outdated lock file. you can update the lock file by running 'poetry update poetry.lock'

File: app/routes.py [hunk 2]
```diff
@@ -9,5 +9,6 @@ def index():
     posts = [
         {"author": {"username": "Sal"}, "body": "Beautiful day in Portland!"},
         {"author": {"username": "Michael"}, "body": "Pyrates are so cool!"},
+        {"author": {"username": "Scott"}, "body": "Ooh, Bootstrap!"},
```
Comment (SalMireles) [inline app/routes.py:12]: nice!

**Review by SalMireles — CHANGES_REQUESTED** [review]
Looks good. Just one request to update the lock file. It would also be helpful to include a before and after image of the changes as a quick reference

**Review by SalMireles — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (Scott-Larsen) [reply pyproject.toml]: I think I've resolved this. Thanks for the heads-up.

File: app/routes.py (hunk 2 above)
Comment (misaacs) [reply app/routes.py:12]: good start :-)

**Review by misaacs — COMMENTED** [review]
Great idea adding bootstrap

**Review by misaacs — CHANGES_REQUESTED** [review]
I couldn't get it to run. On flask run: "Error: Could not import 'app.crypto_app'" ; I made a new branch with a PR into this branch... only change is moving crypto_app.py into app folder

**Review by CloudF1sh — APPROVED** [review]

**Review by SalMireles — CHANGES_REQUESTED** [review]
Requesting to test the app move that was merged into this branch. Please revert if the app was functional before this change

**Comment by Scott-Larsen** [comment]
@SalMireles I feel anything but confident but I'm pretty sure this was working before the change was merged in. My local bootstrap branch still works and when I git checkout origin/feature/scott/#11-add-bootstrap I get an error similar to what @misaacs was getting. With that branch checked out I can `cd` into the app folder and then `flask run` works but I'm guessing `flask run` should work from the base `ppp` folder. There may now be another Poetry conflict but I guess I'd wait to deal with that until after we decide whether or not to roll back the change that was merged into this.

**Comment by SalMireles** [comment]
> @SalMireles I feel anything but confident but I'm pretty sure this was working before the change was merged in. My local bootstrap branch still works and when I git checkout origin/feature/scott/#11-add-bootstrap I get an error similar to what @misaacs was getting. With that branch checked out I can `cd` into the app folder and then `flask run` works but I'm guessing `flask run` should work from the base `ppp` folder. There may now be another Poetry conflict but I guess I'd wait to deal with that until after we decide whether or not to roll back the change that was merged into this.

right, it should run from the top level directory. Maybe @misaacs was in the app package and trying to run from there? We could also update the name to app_run.py so it's clear where to run. Let me post a loom on how to resolve the poetry conflict. I'll think of a way to handle the conflicts better - looks like there is a specific package to handle this, but will look into this some more: https://poetry-merge-lock.readthedocs.io/en/latest/

**Review by misaacs — APPROVED** [review]

**Review by SalMireles — APPROVED** [review]

**Pull request merged**